e2e/deno-e2e/tests/deno.spec.ts

```diff
@@ -113,7 +113,7 @@ Deno.test('Another File', async () => {
       );
     }, 120_000);
 
-    it('should lint deno app w/options', async () => {
+    xit('should lint deno app w/options', async () => {
```

Comment:
`xit` here

Reply:
I had to skip these since they are breaking in CI. Caleb is looking at them. :)